stdlib/3/ast.pyi

```diff
@@ -9,7 +9,7 @@ from typing import overload, Any, Iterator, Optional, Union, TypeVar
 # is imported in both modules. unfortunately we can't just rename sys,
```

Comment:
Can this comment be removed now? It's not clear exactly what it's referring to; there's nothing unusual in the below code.

Reply:
This was introduced in #2887, together with the `# type: ignore` annotation. @msullivan In case this is still relevant, can we reword the comment to reference the `# type: ignore` somehow? As an outsider it's hard for me to judge the relevancy of how `sys` relates to this import or the ignore.

Reply:
I have re-added it for now.